packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -159,47 +160,61 @@ function isPluginDisabled(
   return false
 }
 
+async function loadPluginFactory(
+  remotePlugin: RemotePlugin,
+  obfuscate?: boolean
+): Promise<void | PluginFactory> {
```

Comment:
nit: I think  `Promise<PluginFactory | undefined> ` would be a bit more idiomatic =)